modules/webgl/src/adapter/converters/device-parameters.ts

```diff
@@ -281,35 +281,39 @@ export function setDeviceParameters(device: Device, parameters: Parameters) {
   // COLOR STATE
 
   if (parameters.blendColorOperation || parameters.blendAlphaOperation) {
-    gl.enable(GL.BLEND);
-
-    const colorEquation = convertBlendOperationToEquation(
-      'blendColorOperation',
-      parameters.blendColorOperation || 'add'
-    );
-    const alphaEquation = convertBlendOperationToEquation(
-      'blendAlphaOperation',
-      parameters.blendAlphaOperation || 'add'
-    );
-    gl.blendEquationSeparate(colorEquation, alphaEquation);
+    if (parameters.blendColorOperation === 'none' || parameters.blendAlphaOperation === 'none') {
+      gl.disable(GL.BLEND);
+    } else {
+      gl.enable(GL.BLEND);
```

Comment:
Because neither WebGL nor WebGPU enables blending for color and alpha separately from the other, the answer "what is the default value of `blendAlphaOperation`"  is getting a bit complicated. I wonder if we should:

- (A) disallow 'none' for blendAlphaOperation, only use blendColorOperation to enable/disable blending
- (B) add a separate boolean parameter to enable/disable blending

```javascript
if (parameters.blend === true) {
  const colorEquation = convertBlendOperationToEquation(
    'blendColorOperation',
    parameters.blendColorOperation || 'add'
  );
  // ...
} else {
  // ...
}
```

I lean toward (b), but not a strong preference.

Reply:
While I initially proposed adding a `none` option when I was notified of the problem, after reviewing the PR and thinking more about it, I am also thinking that a separate `blend` enable type parameter seems cleanest.